pkg/restore/db.go

```diff
@@ -129,6 +129,21 @@ func (db *DB) CreateTable(ctx context.Context, table *utils.Table) error {
 			zap.Stringer("table", table.Info.Name),
 			zap.Error(err))
 	}
+
+	// TODO: remove this after tiflash supports restore
+	removeTiFlashSQL := fmt.Sprintf(
+		"alter table %s set tiflash replica 0",
+		utils.EncloseName(tableInfo.Name.O),
+	)
+	err = db.se.Execute(ctx, removeTiFlashSQL)
```

Comment:
what if ignore this error when tidb not support set tiflash replica

Reply:
Would fail, do we need to support the versions of tidb which don't support tiflash?